google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1387,7 +1387,8 @@ public static Builder newBuilder() {
   /**
    * Creates a new bucket.
    *
-   * Accepts an optional userProject {@link BucketTargetOption} option.
+   * Accepts an optional userProject {@link BucketTargetOption} option which defines the project id
```

Comment:
Could you apply this update to all instances of this comment?

Reply:
Right right :)